src/components/App/index.js

```diff
@@ -1,3 +1,13 @@
-import App from './App';
+import React from 'react';
+
+import Item from '../Item/Item';
+
+function App() {
```

Comment:
I guess this is a merge issue. Only `import App from './App';` should stay here.